packages/website/package.json

```
@@ -1,6 +1,11 @@
 {
   "name": "waku-website",
-  "version": "0.1.0",
+  "version": "0.0.0",
```

We don't need to change. It's private anyway though.
```suggestion
  "version": "0.1.0",
```